etc/config/c++.amazon.properties

```diff
@@ -1101,6 +1101,14 @@ libs.gsl.versions.301.version=3.0.1
 libs.gsl.versions.301.path=/opt/compiler-explorer/libs/GSL/v3.0.1/include
 libs.gsl.versions.lite.version=lite
 libs.gsl.versions.lite.path=/opt/compiler-explorer/libs/gsl-lite/include
+libs.entt.name=entt
+libs.entt.versions=trunk:350:360
+libs.entt.versions.350.version=v3.5.0
+libs.entt.versions.350.path=/opt/compiler-explorer/libs/entt/v3.5.0/src
```

Comment:
should this be the `single_include` directory? Looks like either will work - merging as is but let me know if it needs to be pointed elsewhere

Reply:
I prefer the src directory, for some better error messages (line 31 in any.hpp seems better than line 1520 in entt.hpp)

both work with `#include <entt/entt.hpp>` all the same.